specification/resources/spaces/key_create.yml

```
@@ -0,0 +1,75 @@
+operationId: spacesKey_create
+
+summary: "[Beta] Create a New Spaces Access Key"
```

```suggestion
summary: "Create a New Spaces Access Key"
```

This is being released directly to GA. We should remove the beta tags here and elsewhere.